server/athenian/api/auth.py

```diff
@@ -273,4 +275,11 @@ def _get_token_auth_header(self, request: aiohttp.web.Request) -> str:
         except Exception:
             return aiohttp.web.Response(body="Your auth token is likely revoked.", status=401)
 
+        god = await request.sdb.fetch_one(select([God.mapped_id]).where(God.user_id == user.id))
+        if god is not None:
+            mapped_id = god[God.mapped_id.key]
+            if mapped_id is not None:
+                user.god_id = user.id
+                user.id = (await self.get_users([mapped_id]))[0]
```

Comment:
Why not simply as follows?
```suggestion
                user.id = mapped_id
```

Reply:
You found a bug haha. It should be like this:
```python
god_id = user.id
user = (await self.get_users([mapped_id]))[0]
user.god_id = god_id
```